templates/cassiopeia/index.php

```diff
@@ -166,57 +166,59 @@
 		<?php endif; ?>
 	</header>
 
-	<?php if ($this->countModules('banner', true)) : ?>
-		<div class="container-banner full-width">
-			<jdoc:include type="modules" name="banner" style="none" />
-		</div>
-	<?php endif; ?>
+    <div class="site-grid">
```

Comment:
```suggestion
	<div class="site-grid">
```
Fix code style error which causes drone to fail.

Please use tabs and not spaces to indent PHP code.

Reply:
@richard67 sorry. Pushed fix.